src/tribler-common/tribler_common/sentry_reporter/tests/test_sentry_tools.py

```diff
@@ -0,0 +1,80 @@
+from tribler_common.sentry_reporter.sentry_tools import first, last, modify, parse_os_environ, parse_stacktrace, \
+    safe_delete, safe_get
+
+
+def test_first():
+    assert first(None, '') == ''
+    assert first([], '') == ''
+    assert first(['some'], '') == 'some'
+    assert first(['some', 'value'], '') == 'some'
+
+    assert first((), '') == ''
+    assert first(('some', 'value'), '') == 'some'
+
+    assert first(None, None) is None
+    assert first(range(5)) == 0
+
+
+def test_last():
+    assert last(None, '') == ''
+    assert last([], '') == ''
+    assert last(['some'], '') == 'some'
+    assert last(['some', 'value'], '') == 'value'
+
+    assert last((), '') == ''
+    assert last(('some', 'value'), '') == 'value'
+    assert last(range(5)) == 4
```

Comment:
Do we really have to test Python itself :wink:

Reply:
I mean, these are trivial helper one-liners that are intended to make the code *shorter* and *more readable*. What is the probability of someone *changing* them in a way that could break these test? Why would anyone want to do that?

There is literally no complex logic in these one-liners, there is *nothing* to break. Python is a mature language, we trust its developers not to break the basic conventions of the language (e.g. iteration). And as long as those conventions hold these tests *cannot* fail.

Reply:
>Do we really have to test Python itself

It has been added after renaming `l`->`iterable`. Not needed now.

Reply:
Fixed